src/Views/ConfirmationSuccess.js

```diff
@@ -1,14 +1,20 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+import queryString from 'query-string'
+import axios from 'axios'
 import { Link } from 'react-router-dom'
 import styled from 'styled-components'
+
+import Spinner from '../Components/Spinner'
 import Success2 from '../Assets/success1.svg'
+import Error from '../Assets/error.svg'
+
+const URL = 'http://localhost:9000/confirm'
```

Comment:
rather than hard code this in here can we have a file that sets axois baseURL and have that url passed as env variable